src/base/base.c

```diff
@@ -635,7 +644,7 @@ void xdebug_base_rinit()
 	XG_BASE(filters_tracing)           = xdebug_llist_alloc(xdebug_llist_string_dtor);
 	XG_BASE(filters_code_coverage)     = xdebug_llist_alloc(xdebug_llist_string_dtor);
 
-	/* Overload var_dump, set_time_limit, error_reporting, and pcntl_exec */
+	/* Overload var_dump, set_time_limit, error_reporting, pcntl_exec and pcntl_fork */
```

Comment:
Just remove this line, it doesn't say anything any more (After the big refactoring), and keeping it up to date is just annoying.